index.html

```diff
@@ -41,8 +41,122 @@ <h3 class="link-1"> <a href="#"> LET'S CONNECT</a></h3>
                    </ul>
                 </div>
             </footer>
+        </section> 
+        </div>
+        <section>
```

Comment:
- [ ] Kindly make sure to use `display:grid`  in your projects container as specified in the project requirements.